frontend/src/types/api.d.ts

```diff
@@ -28,7 +28,7 @@ declare namespace Language {
     id: number;
     audio_exclude: PythonBoolean;
     forced: PythonBoolean;
-    hi: PythonBoolean;
+    hi: string;
```

Comment:
Would be great if you can narrow down the types here:

```suggestion
    hi: 'never' | 'also' | 'only';
```